examples/guaranteed-delivery/Ballerina.toml

```
@@ -0,0 +1,8 @@
+[package]
+org = "ballerina"
```

I think we use `wso2` as the organization for the examples?

```suggestion
org = "wso2"
```

---

Fixed via [28d6030](https://github.com/ballerina-platform/module-ballerina-messaging/pull/2/commits/28d6030d857c63b64a1e954f49566cd2756dec7e)